### PR TITLE
Add robots.txt and sitemap

### PR DESCRIPTION
**Pull request by mruwnik**
https://github.com/StampyAI/stampy-ui/issues/263

### CODE REVIEW

File: app/routes/[robots.txt].ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  // set up our text content that will be returned in the response
+  const origin = new URL(request.url).origin
+  const robotText = `
+User-agent: Googlebot
+Disallow: /nogooglebot/
+
+User-agent: *
+Allow: /
+
+Sitemap: ${origin}/sitemap.xml
```
Comment (mruwnik) [inline app/routes/[robots.txt].ts:14]: is there a better way to create this file? it's static other than this one url, which really only needs the origin

File: app/routes/[sitemap.xml].tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const formatQuestion = ({pageid, updatedAt}: Question) => `
+    <url>
+      <loc>${origin}/?state=${pageid}</loc>
+      <lastmod>${updatedAt}</lastmod>
+      <priority>1.0</priority>
+    </url>
+    `
+  const {data} = await loadAllQuestions(request)
+
+  const urls = data.filter((q) => q.status == QuestionStatus.LIVE_ON_SITE).map(formatQuestion)
```
Comment (mruwnik) [inline app/routes/[sitemap.xml].tsx:15]: I'm assuming that only live questions should be indexed...

File: app/routes/[robots.txt].ts (hunk 1 above)
Comment (Aprillion) [reply app/routes/[robots.txt].ts:14]: I would say `https://aisafety.info/sitemap.xml` since we don't want to index any other domains 😅

Comment (mruwnik) [reply app/routes/[robots.txt].ts:14]: what about stampy.ai?

Comment (mruwnik) [reply app/routes/[robots.txt].ts]: Perfect! That makes things a lot simpler

**Review by Aprillion — APPROVED** [review]

**Pull request merged**